src/Robo/Plugin/Commands/DevelopmentModeCommands.php

```diff
@@ -77,12 +116,219 @@ public function devRefreshAll(
             $result = $this->devRefreshDrupal(
                 environmentType: LocalDevEnvironmentTypes::from($environmentType),
                 siteName: $siteName,
-                startLocalEnv: $startLocalDev,
             );
         }
         return $result;
     }
 
+    /**
+     * Refresh a site database in DDEV.
```

Comment:
If we are moving to a model where there is only one supported local development model (ie DDEV), then I feel I would want to refactor this either as `databaseRefreshLocal` (removing implementation details from the name of the function), or, better, by defining an interface with `databaseRefresh()` and then a base class and a class for each environment type (tugboat, local etc) that is loaded by a factory depending on environment.

Reply:
I was inclined to leave the idea of supporting multiple environment types since it is already structured/written and who knows what the future holds. I was trying to avoid the pain of rewriting half of this 18 months from now. 

If we want to discuss excising even the _idea_ of supporting multiple local environments, maybe that can be its own ticket? How strongly do you feel about this?

Reply:
> excising even the idea of supporting multiple local environments

I wasn't feeling strongly about it, but seeing the strong push to remove Lando everywhere made me think one way to dev was already the policy, so I was suggesting that the code no longer needed to have "flavors" in its function signatures.